ceph-dashboard-nightly/config/definitions/ceph-dashboard-nightly.yml

```diff
@@ -0,0 +1,43 @@
+- job:
+    name: ceph-dashboard-nightly
+    project-type: freestyle
+    defaults: global
+    concurrent: true
+    node: huge && x86_64 && centos7 && rebootable
+    display-name: 'ceph: dashboard nightly'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          num-to-keep: 300
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph/
+      - rebuild:
+          auto-rebuild: true
+
+    triggers:
+      - timed: '@daily'
+
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph.git
+          branches:
+            - origin/master
+            - origin/nautilus
+          browser: auto
+          timeout: 20
+          skip-tag: true
+          shallow-clone: true
+          wipe-workspace: true
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../../scripts/build_utils.sh
+            - ../../build/build
+      - shell: "cd src/pybind/mgr/dashboard; timeout 7200 ./run-backend-api-tests.sh"
```

Comment:
Should we run the dashboard e2e tests as well?

Reply:
Sure. it makes sense.